pkg/dao/repository_configs.go

```diff
@@ -43,11 +43,11 @@ func DBErrorToApi(e error) *ce.DaoError {
 	if ok {
 		if pgError.Code == "23505" {
 			switch pgError.ConstraintName {
-			case "repo_and_org_id_unique":
+			case "repo_config_repo_org_id_deleted_null_unique":
 				dupKeyName = "URL"
 			case "repositories_unique_url":
 				dupKeyName = "URL"
-			case "name_and_org_id_unique":
+			case "repo_config_name_deleted_org_id_unique":
```

Comment:
I was contemplating removing this test because it only really tests the implementation of the `DBErrorToAPI` function. It didn't actually catch this bug. Thoughts?

Reply:
did you mean to point to specific test?

Reply:
yeah whoops, I meant here: https://github.com/content-services/content-sources-backend/blob/af65a0279e05604169e35b3fe08760fd01b460f8/pkg/dao/repository_configs_test.go#L1559

Reply:
IDK, i feel like it still has some value?

Reply:
oh, i see what you're saying, yeah, i think i we have tests that test the behavior and the error response, that is a lot more valuable.

Reply:
I'd say lets beef up TestRepositoryCreateAlreadyExists  to check the message, and make sure a test for name & url is covered, and then remove TestDBErrorToApi

Reply:
I already added a test to TestRepositoryCreateAlreadyExists. Are you saying I should add something else to it?

Reply:
yes, i meant testing the actual message, similar to what TestDBErrorToApi is doing.  This would help tell the difference if its name vs url instead of just checking for daoError.BadValidation)

Reply:
and the check could be something like    message.includes?("name")  instead of doing a literal string comparison.

Reply:
sorry, maybe i should have drank more coffee :)   Feel free to delete that test, otherwise ACK!

Reply:
ha, no worries :)